src/modules/user-segment/user-segment.controller.spec.ts

```diff
@@ -5,16 +5,151 @@ import { UserSegmentService } from './user-segment.service';
 describe.skip('UserSegmentController', () => {
```

Comment:
```suggestion
describe('UserSegmentController', () => {
```
`.skip` was just added to skip failing test, it can be removed whenever you make a PR